build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "sangria"
 organization := "org.sangria-graphql"
-version := "1.4.3-SNAPSHOT"
+version := "1.4.4-SNAPSHOT"
```

Comment:
Should this be bumped to `1.5.0-SNAPSHOT` since the signature of the case class `ProjectedName` changed, which is potentially a breaking API change for users of that class?